tests/periph_spi/main.c

```diff
@@ -99,6 +99,38 @@ static uint32_t _xtimer_diff_usec(xtimer_ticks32_t stop, xtimer_ticks32_t start)
     return xtimer_usec_from_ticks(xtimer_diff(stop, start));
 }
 
+static int _print_arch_help(void)
+{
+#ifdef CLOCK_APB1 /* STM32 */
+#define HAS_ARBITRARY_BUS_SPEED
+    int clk, tmp;
+
+    for (int i = 0; i < (int)SPI_NUMOF; i++) {
+        printf("\t\tSPI_DEV(%i), ", i);
+        if (spi_config[i].apbbus == APB1) {
+            clk = CLOCK_APB1;
+            printf("CLOCK_APB1 = %luHz\n", (uint32_t)CLOCK_APB1);
+        }
+        else if (spi_config[i].apbbus == APB2) {
+            clk = CLOCK_APB2;
+            printf("CLOCK_APB2 = %luHz\n", (uint32_t)CLOCK_APB2);
+        }
+        else {
+            printf("error: apbbus configuration ");
```

Comment:
```suggestion
            puts("error: apbbus configuration ");
```

Reply:
Sorry but I gave up this part of the code, anyway thanks for the review.